app/controllers/articles_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ArticlesController < ApplicationController
 	#Test comment for Hound	
+	#Test comment2 for Hound
```

Comment:
Will be fixed later


-- Vicky Kumar Singh


On Tue, Apr 12, 2016 at 12:18 PM, Hound <notifications@github.com> wrote:

> In app/controllers/articles_controller.rb
> <https://github.com/Dextostone/sc_ruby/pull/2#discussion_r59327772>:
>
> > @@ -1,5 +1,6 @@
> >  class ArticlesController < ApplicationController
> >  	#Test comment for Hound	
> > +	#Test comment2 for Hound
>
> Missing space after #.
> Tab detected.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Dextostone/sc_ruby/pull/2/files/39a4e97653abc778b9a2be87cc21b2654eb52291#r59327772>
>